src/org/parosproxy/paros/control/MenuFileControl.java

```diff
@@ -453,30 +455,14 @@ public String getDescription() {
     		}
     		
     		try {
-	    	    waitMessageDialog = view.getWaitMessageDialog(Constant.messages.getString("menu.file.savingSession"));	// ZAP: i18n
+	    	    waitMessageDialog = view.getWaitMessageDialog(Constant.messages.getString("menu.file.savingSnapshot"));	// ZAP: i18n
 	    	    control.snapshotSession(fileName, this);
         	    log.info("snapshot as session file " + session.getFileName());

```

Comment:
Just noticed that this should use `fileName` (instead of `session.getFileName()`). Sorry, for noticing sooner.